DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/SwiftUI/DisplayList+Reflection.swift

```diff
@@ -152,8 +152,12 @@ extension DisplayList.Content.Value: Reflection {
         case let (.enum("image"), image):
             self = try .image(reflector.reflect(image))
 
-        case (.enum("drawing"), _):
-            self = .unknown
+        case let (.enum("drawing"), (contents, origin, _) as (NSObject, CGPoint, Any)):
+            if let image = GraphicsImage(rasterizing: contents, origin: origin) {
+                self = .image(image)
```

Comment:
suggestion/ The rendering will occur on the processing thread as we are applying a lazy reflection. But this operation is implicit and the fact that we map `.enum("drawing")` to a `.image` content kind of break the reflection pattern.

I would suggest to map this to a new `self = .drawing(...)` case and do the rendering in the `SwiftUIWireframesBuilder` when mapping the `DisplayList.Content` to a `SRWireframe.image`. This way we are explicitly rendering the `.drawing` content to an `.image` wireframe on the processing thread. WDYT?

Reply:
Yes, that makes sense. I will move the rasterization where you suggest. Thanks!